static/app/views/externalRedirect/index.tsx

```diff
@@ -0,0 +1,100 @@
+import {Component} from 'react';
+import styled from '@emotion/styled';
+
+import Panel from 'sentry/components/panels/panel';
+import {tct} from 'sentry/locale';
+
+interface ExternalRedirectState {
+  count: number;
+  redirectUrl: string | null;
+}
+
+class ExternalRedirect extends Component<{}, ExternalRedirectState> {
```

Comment:
For new components, we shouldn't use classes. Could you rewrite this as a functional component using hooks?